tests/network-tests/run-migration-tests.sh

```diff
@@ -1,87 +1,194 @@
 #!/usr/bin/env bash
 set -e
 
-# THIS IS JUST A REFERENCE SCRIPT, NOT USED FOR ACTUAL GIZA->OLYMPIA TESTING
-
 SCRIPT_PATH="$(dirname "${BASH_SOURCE[0]}")"
 cd $SCRIPT_PATH
 
-# The joystream/node docker image tag which contains WASM runtime to upgrade chain with
-TARGET_RUNTIME_TAG=${TARGET_RUNTIME_TAG:=$(../../scripts/runtime-code-shasum.sh)}
-# The joystream/node docker image tag to start the chain with
-RUNTIME_TAG=${RUNTIME_TAG:=6740a4ae2bf40fe7c670fb49943cbbe290277601}
-# Post migration assertions by means of typescript scenarios required
-POST_MIGRATION_ASYNC_ASSERTIONS=${POST_MIGRATION_ASYNC_ASSERTIONS:=$true}
-# source common function used for node setup
-source ./node-utils.sh
+# Log only to stderr
+# Only output from this script should be the container id of the node at the very end
+
+# Location that will be mounted to /spec in containers
+# This is where the initial balances files and generated chainspec files will be located.
+DATA_PATH=$PWD/data
+mkdir -p ${DATA_PATH}
+
+# The latest docker image tag to use for joystream/node (testing profile)
+RUNTIME=${RUNTIME:=mainnetDev}
```

Comment:
if we can keep the names of the runtimes general would be better. mainnet doesn't make sense, but maybe "current" and "target".. something that works when this code is on a future branch post ephesus .. and with the comment "The latest docker image tag" it is even more confusing.